services/QuillLMS/app/models/activity.rb

```diff
@@ -289,10 +292,28 @@ def segment_activity
     is_evidence? && saved_change_to_name?
   end
 
+  private def update_evidence_flags?
+    is_evidence? && saved_change_to_flags? && child_activity
+  end
+
   private def update_evidence_child_title
     child_activity&.update(title: name)
   end
 
+  private def log_evidence_flag_change
+    change_log = {
```

Comment:
I would get rid of the temp variable `change_log` and call `create` on hash directly:

```
Change.create(
  user_id: @lms_user_id,
  action: ChangeLog::EVIDENCE_ACTIONS[:update],
  ..
)
```